src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -1,43 +1,139 @@
 package cromwell.engine
 
-import akka.actor.Actor
+import akka.actor.{ActorRef, Props, Actor}
 import akka.actor.SupervisorStrategy.Stop
+import akka.event.LoggingReceive
+import cromwell.binding.{FullyQualifiedName, Workflow, WdlBinding, WdlValue}
 
 /**
  * This model assumes workflow actors are transient, created once per workflow instance.
  */
 object WorkflowActor {
```

Comment:
can you add a sealed trait called something like WorkflowActorMessage and then have all of these extend that